guides/10_other-tutorials/building-an-mcp-client-with-gradio.md

```diff
@@ -0,0 +1,489 @@
+# Using the Gradio Chatbot as an MCP Client
+
+This guide will walk you through a Model Context Protocol (MCP) Client and Server implementation with Gradio. You'll build a Gradio chatbot that can respond as an LLM (using Anthropic's Claude API) as well as generate images (thanks to a separate Gradio MCP Server).
+
+<video src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/gradio-guides/mcp-guides.mp4" style="width:100%" controls preload> </video>
```

Comment:
This url does not actually load for me

Reply:
For some reason not loading directly for me (maybe because it's a large file?) but tested on a Markdown viewer and it does load inline in the browser:

<img width="711" alt="image" src="https://github.com/user-attachments/assets/e69c72d0-90ae-47cc-9010-d5be96a0a2fd" />